maps/torch/items/clothing/solgov-head.dm

```diff
@@ -30,6 +30,16 @@
 	desc = "It's a black ballcap bearing the Expeditonary Corps crest. The brim has gold trim."
 	icon_state = "expeditioncomsoft"
 
+/obj/item/clothing/head/soft/solgov/fleet/hijab
+	name = "fleet hijab"
+	desc = "It's a veil which is wrapped to cover the head and chest.  This one is tighter fitting and has a Fleet insignia embroidered into it."
+	icon_state = "hijab_fleet"
+
+/obj/item/clothing/head/soft/solgov/fleet/turban
+	name = "fleet turban"
+	desc = "It's a sturdy cloth, worn around the head.  This one is tighter fitting and has a fleet insignia embroidered into it."
```

Comment:
`fleet` -> `Fleet`

Reply:
Typo fixed.  I accidentally made another commit but that's squashed and since it just changed one letter the checks should all be fine